src/@next/components/molecules/ProductDescription/ProductDescription.tsx

```diff
@@ -49,8 +48,8 @@ export const ProductDescription: React.FC<IProps> = ({
         </S.TabTitle>
       </S.Tabs>
       <div hidden={activeTab !== TABS.DESCRIPTION}>
-        {descriptionJson ? (
-          <RichTextEditorContent jsonData={descriptionJson} />
+        {description ? (
+          <RichTextEditorContent jsonData={description} />
         ) : (
           <p>{description}</p>
         )}
```

Comment:
```suggestion
        {description && (
          <RichTextEditorContent jsonData={description} />
        )}
```
Current check doesn't make sense 🤷